read_zdf_vis_3d.py

```diff
@@ -0,0 +1,49 @@
+"""
+Import ZDF point cloud and visualize it using vtk_visualizer (point cloud) and
+matplotlib (RGB image and Depth map).
```

Comment:
This docstring is confusing to read. Terminology is not used accurately

Reply:
Changed to 'Import ZDF point cloud and visualize it.'
The libs are probably not too relevant for the description.